pubkey_encrypt.module

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Implements hook_form_alter().
+ */
+function pubkey_encrypt_form_alter(&$form, $form_state, $form_id) {
+  switch ($form_id) {
+    case 'user_login_form':
+      // Try to have this form pass through our custom handler first.
+      array_unshift($form['#submit'], 'custom_login_submit');
```

Comment:
"custom_login_submit" ??????

This should be "pubkey_encrypt_login_submit" or removed.